python/tk_hiero_export/sg_shot_processor.py

```diff
@@ -95,11 +96,64 @@ def populateUI(self, widget, exportItems, editMode=None):
         shotgun_layout.addWidget(collating_widget)
         CollatingExporterUI.populateUI(self, collating_widget, properties)
 
+        if _cutsSupported(self.app.shotgun):
```

Comment:
I had it as part of the Processor class originally, but then I needed it here in the ProcessorUI class so I moved it out.